Plan/common/src/main/java/com/djrapitops/plan/delivery/rendering/json/graphs/pie/PieGraphFactory.java

```diff
@@ -68,6 +68,10 @@ public Pie serverPreferencePie(Map<String, Long> serverPlaytimes) {
         return new ServerPreferencePie(serverPlaytimes);
     }
 
+    public Pie hostnamePie(Map<String, Integer> hostname) {
+        return new hostnamePie(hostname);
```

Comment:
Class names should be PascalCase (Refactor rename)
```suggestion
        return new HostnamePie(hostname);
```